src/hooks/useQuickbooks/useQuickbooks.ts

```diff
@@ -27,7 +27,8 @@ export const useQuickbooks: UseQuickbooks = () => {
   // Poll the server to determine when the Quickbooks sync is complete
   useEffect(() => {
     if (isSyncingFromQuickbooks && syncStatusIntervalRef.current === null) {
-      const interval = window.setInterval(() => fetchIsSyncingFromQuickbooks(), 2000)
+      const interval = setInterval(
+        () => void fetchIsSyncingFromQuickbooks(), 2000) as unknown as number
```

Comment:
I was getting typescript error before:

![image](https://github.com/user-attachments/assets/ad022cd1-c190-434e-a8df-2c79203f5aa3)

Not sure if does it happen only to me..